trie-root/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "trie-root"
-version = "0.15.2"
+version = "0.16.0"
```

Comment:
i this this bump is not necessary as `reference-trie` is a dev-dependency, but let's be on the safer side